### PR TITLE
[bug 758535] fixes static file handling

### PR DESCRIPTION
**Pull request by readevalprint**
Serves media files from the recommended folders. Uses django compressor.
Also adds some debugging tools such as command extensions and debug toolbar.

### CODE REVIEW

File: project/settings/base.py [hunk 1]
```diff
@@ -13,36 +13,36 @@
...
-    }
-}
-
 # Defines the views served for root URLs.
 ROOT_URLCONF = '%s.urls' % PROJECT_MODULE
 
-INSTALLED_APPS = list(INSTALLED_APPS) + [
+INSTALLED_APPS = (
+    'funfactory',
+    'jingo_minify',
```
Comment (jsocol) [inline project/settings/base.py]: If we're using `compressor` we should probably drop this?

File: project/settings/base.py [hunk 2]
```diff
@@ -13,36 +13,36 @@
...
+    'cronjobs',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.staticfiles',
+    'django.contrib.sessions',
+    'commonware.response.cookies',
+    'djcelery',
+    'django_nose',
+    'session_csrf',
+    'product_details',
```
Comment (jsocol) [inline project/settings/base.py]: I don't think we need `product_details` at all, do we?

File: project/settings/base.py [hunk 3]
```diff
@@ -85,6 +86,43 @@
...
+MEDIA_URL = r'media'
+
+# Templates
+TEMPLATE_LOADERS = (
+    'jingo.Loader',
+    'django.template.loaders.filesystem.Loader',
+    'django.template.loaders.app_directories.Loader',
+    )
+
+TEMPLATE_DIRS = here('templates')
```
Comment (jsocol) [inline project/settings/base.py]: Does this need to be a tuple?

File: project/settings/base.py [hunk 4]
```diff
@@ -85,6 +86,43 @@
...
+    'extensions': [
+    'tower.template.i18n',
+    'jinja2.ext.do',
+    'jinja2.ext.with_',
+    'jinja2.ext.loopcontrols',
+    'compressor.contrib.jinja2ext.CompressorExtension',
+    ]
+}
+
+AUTHENTICATION_BACKENDS = (
```
Comment (jsocol) [inline project/settings/base.py]: I'd move this below `TEMPLATE_CONTEXT_PROCESSORS` just because it keeps the templaty stuff together.

Comment (readevalprint) [inline project/settings/base.py]: sure

File: project/settings/base.py (hunk 1 above)
Comment (readevalprint) [inline project/settings/base.py]: right

File: project/settings/base.py (hunk 2 above)
Comment (readevalprint) [inline project/settings/base.py]: haha, this is the first thing that was broken on the new playdoh app. I wasn't really sure of it's purpose.

File: project/settings/base.py (hunk 3 above)
Comment (readevalprint) [inline project/settings/base.py]: yup

**Comment by readevalprint** [comment]
r?

**Pull request closed without merging**